test/tests/functional/pbs_config.py

```diff
@@ -81,7 +81,7 @@ def test_config_as_snap(self):
         # Now that we have a custom configuration, take a snapshot
         outdir = pwd.getpwnam(self.du.get_current_user()).pw_dir
         snap_cmd = [pbs_snapshot_path, "-o " + outdir, "--with-sudo"]
-        ret = self.du.run_cmd(cmd=snap_cmd, logerr=False)
+        ret = self.du.run_cmd(cmd=snap_cmd, logerr=False, as_script=True)
```

Comment:
Just curious, why do you need as_script=True?  You are just running a normal command.  You aren't doing anything that requires the shell the parse it.

Reply:
it fails without it. I think it's because we run some commands with sudo when --with-sudo is provided.